index.js

```diff
@@ -65,8 +65,9 @@ module.exports = function (options) {
 
 function toLink(pkg, num, words) {
   var res = '';
-  res += link(pkg.name, pkg.homepage);
-  res += truncate(pkg.description, pkg.homepage, words);
+  var homepage = pkg.homepage.split('#readme').join('')
```

Comment:
and I know I asked you to change it before!!! I wasn't thinking about this downside before, so you were right the first time! sorry, and thank you! lol